core/includes/file.inc

```diff
@@ -536,6 +536,9 @@ function file_save_htaccess($directory, $private = TRUE, $force_overwrite = FALS
  */
 function file_htaccess_lines($private = TRUE) {
   $lines = <<<EOF
+# This code block is replicated in the file_htaccess_lines() function in
+# /core/includes/file.inc and files/.htaccess. If you update this file then 
```

Comment:
There is a blank space at the end of this line, and also if/then in sentences should be separated with a comma:

```suggestion
# /core/includes/file.inc and files/.htaccess. If you update this file, then
```